ipv-stub-template.yml

```diff
@@ -65,6 +65,13 @@ Resources:
 
   IPVAuthorizeLambda:
     Type: AWS::Serverless::Function
+    Connectors:
```

Comment:
Could we use something like this instead?
https://github.com/govuk-one-login/authentication-api/blob/main/template.yaml#L4187